backend/src/utils/validationRules.js

```
@@ -1,54 +1,55 @@
+   },
+   USER: {
+      USER_NOT_FOUND: "User not found",
+      INVALID_CREDENTIALS: "Invalid credentials",
+      REQUIRED_FIELDS: "Required fields are missing",
+      FIELD_REQUIRED: FIELD_REQUIRED,
+      INVALID_REFERRER: "Invalid referer URL",
+   },
+   SESSION: {
+      INVALID_SESSION: "Invalid session",
+      SESSION_EXPIRED: "Session expired",
+      SESSION_DESTROYED: "Session destroyed",
```

Biome (1.9.4)

[error] This property value named SESSION_DESTROYED is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named SESSION_DESTROYED


(lint/suspicious/noDuplicateObjectKeys)